package.json

```diff
@@ -31,8 +31,7 @@
     "classnames": "^2.2.5",
     "material-components-web": "^0.27.0",
     "react": "^16.2.0",
```

Comment:
@jrajav can you do me a favor and also remove react and react-dom from dependencies? They made it in there by accident. These are covered in peer dependencies, so they don't have to be moved, you can just safely delete them.

Reply:
@jamesmfriedman Done!